src/main/java/gov/usgs/wma/waterdata/groundwater/BuildRdbFile.java

```diff
@@ -88,12 +88,19 @@ protected ResultObject processRequest(String locationFolder) {
 			RdbWriter rdbWriter = createRdbWriter(writer).writeHeader();
 			dao.sendDiscreteGroundWater(states, rdbWriter, aqDao.getParameters());
 
+			if (rdbWriter.getDataRowCount() == 0) {
```

Comment:
Can we add a comment just above this if block briefly explaining why we don't want to send the file to s3?

Reply:
I think a separate comment wouldn't be needed if the RuntimeException was more descriptive.  What if the RE mentions that this is a "rule violation as described in JIRA IOW-728".... Or something similar.